.github/workflows/validate.yml

```diff
@@ -0,0 +1,28 @@
+name: Validate code 
```

Comment:
maybe going too far, but is there a way we can share the common yaml between workflows?

Reply:
Not that i know of, you can probably move some stuff into a third workflow that just does the boiler plate, but it feels a bit like overkill.

Otherwise we can combine them into a single workflow file :)